app/src/main/java/com/epam/caloriecalc/data/remote/RetrofitHelper.kt

```diff
@@ -0,0 +1,15 @@
+package com.epam.caloriecalc.data.remote
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitHelper {
+
+    const val baseUrl = "https://benchfoodapp.com"
```

Comment:
There is a Constants file in the util package, I think we should keep them together

Reply:
I would actually disagree here, if a constant is strictly related to a class, object or any other structure it's a good idea to keep it there, otherwise the constants file can get quite bloated, imo all constants have a nice place to be in the existing files without the necessity of a dedicated file, but this is just my personal preference